debezium-core/src/main/java/io/debezium/relational/TableSchema.java

```diff
@@ -55,23 +56,33 @@
     /**
      * Create an instance with the specified {@link Schema}s for the keys and values, and the functions that generate the
      * key and value for a given row of data.
-     * 
+     *
+     * @param parentSchemaName the name of the schema; may be null
```

Comment:
Maybe [this section of the MySQL documentation](http://debezium.io/docs/connectors/mysql/#change-events-value) helps a bit??